pkg/glance/const.go

```diff
@@ -59,12 +62,32 @@ const (
 	// LogVolume is the default logVolume name used to mount logs on both
 	// GlanceAPI and the sidecar container
 	LogVolume = "logs"
+
 	//DBPurgeAge -
 	DBPurgeAge = 30
+	//DBPurge -
+	DBPurge CronJobType = "purge"
+	//CacheCleaner -
+	CacheCleaner CronJobType = "cleaner"
+	//CachePruner -
+	CachePruner CronJobType = "pruner"
 	//DBPurgeDefaultSchedule -
 	DBPurgeDefaultSchedule = "1 0 * * *"
+	//CacheCleanerDefaultSchedule -
+	CacheCleanerDefaultSchedule = "1 0 * * *"
+	//CachePrunerDefaultSchedule -
+	CachePrunerDefaultSchedule = "*/30 * * * *"
 )
 
+// DBPurgeCommandBase -
+var DBPurgeCommandBase = [...]string{"/usr/bin/glance-manage", "--debug", "--config-dir /etc/glance/glance.conf.d", "db purge "}
```

Comment:
Is it intentional to have the extra space at the end of `"db purge "`?  I see it was there beforehand in the `cronjob.go` file, but just want to make sure.

Reply:
Yeah, it was intentional, because then I can just append the `DBPurgeAge` int:

```
 ( cronJobCommand = append(cronJobCommand, strconv.Itoa(DBPurgeAge))
 ```
 
and not worry about dealing w/ extra spaces when I turn it into a cmd string with:

```
commandString := strings.Join(cronCommand, " ")
```

It basically makes the logic clean enough, and even if a `DBPurgeAge` int is not provided
(resulting in an empty string), the command will work